content/auth/index.textile

```diff
@@ -0,0 +1,80 @@
+---
+title: Authentication
+meta_description: "Ably supports two main authentication schemes: basic authentication and token authentication. Token authentication can be implemented using JWTs, Ably tokens, and Ably token requests."
+redirect_from:
+  - /core-features/authentication
+  - /general/authentication
+  - /general/_authentication_capabilities
+---
+
+Before a client or server can issue requests to Ably, such as subscribe to channels, or publish messages, it must authenticate with Ably. Authentication requires an Ably API key. 
+
+h2(#api-keys). Ably API keys
+
+Every Ably app can have one or more API keys associated with it in order to authenticate directly with Ably, or to issue tokens with. API keys can be created with different "capabilities":/auth/capabilities and any tokens issued using that API key can only permit a subset of those capabilities.
+
+h3(#format). API key format
+
+An Ably API key string has the following format: @I2E_JQ.OqUdfg:EVKVTCBlzLBPYJiCZTsIW_pqylJ9WVRB5K9P19Ap1y0@.  
+
+The API key is made up of three parts:
+
+1. @I2E_JQ@ is the public app ID (the part before the first period)
+2. @OqUdfg@ is the public app key ID (the part after the period and before the colon). @I2E_JQ.OqUdfg@ is the public API key ID (both the public app ID and app key ID together)
+3. @EVKVTCBlzLBPYJiCZTsIW_pqylJ9WVRB5K9P19Ap1y0@ is the API key secret and should never be shared with untrusted parties (the part after the colon)
+
+The API key string that is your private secret and should never be made public. This API key string is used in all of our SDKs and for authentication with the REST API.
```

Comment:
"The API key string that is your private secret and should never be made public."

The API key string is your private secret and should never be made public.

Reply:
Reworded this to make it more legible - good spot.